combat-trainer.lic

```diff
@@ -352,6 +355,12 @@ class LootProcess
     end
   end
 
+  def should_perform_ritual(game_state)
+    unless game_state.prepare_consume
```

Comment:
Unless confuses me too.  I'm going to remove that with the simplified logic suggested above.

I think we should stick with prepare_consume as well.  That's what the old code was using and I haven't heard of any errors/issues so might be best to keep that part consistent.